src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('x is %d, y is %g, z is %s' % (x, y, z))
```

Comment:
You might want to check out some formatting options with floats to make your output more accurately reflect what's being asked for